traffic-dashboard/src/pages/about/about.tsx

```diff
@@ -0,0 +1,160 @@
+import { ArrowRight } from "@mui/icons-material";
+import { Card, IconButton } from "@mui/material";
+import React, { ReactElement } from "react";
+import { useNavigate } from "react-router-dom";
+import { routes } from "../../constants/routes";
+
+const Number: React.FC<{ number: number }> = ({ number }) => {
+  return (
+    <div className="flex-2 flex flex-col p-5 justify-center">
+      <div className="w-8 h-8 font-bold text-gray-700 rounded-full bg-white flex items-center justify-center font-mono">
+        {number}
+      </div>
+    </div>
+  );
+};
+
+interface IGettingStartedCardProps {
+  title: string;
+  content: ReactElement;
+  number: number;
+  action?: () => void;
+}
+
+const GettingStartedCard: React.FC<IGettingStartedCardProps> = ({
+  title,
+  content,
+  number,
+  action,
+}) => {
+  return (
+    <>
+      <Card className="h-full">
+        <div className="py-2 flex h-full">
+          <Number number={number} />
+          <div className="">
+            <div className="font-bold text-l mb-2">{title}</div>
+            <p className="text-gray-400 text-base text-sm">{content}</p>
+          </div>
+          <div className="flex-1 flex flex-col px-5 justify-center w-full items-end">
+            {action && (
+              <div>
+                <IconButton onClick={action}>
+                  <ArrowRight />
+                </IconButton>
+              </div>
+            )}
+          </div>
+        </div>
+      </Card>
+    </>
+  );
+};
+
+export const About = () => {
+  const navigate = useNavigate();
```

Comment:
Hey, just read about the useNavigate hook. 
The react-router documentation recommends using redirect() instead. 
https://reactrouter.com/en/main/hooks/use-navigate

Reply:
It's better to use in specific use cases. Thanks for letting me know about this function.